File: Difficulty: Easy/Missing in Array/missing-in-array.py
#User function Template for python3
class Solution:
    
    # Note that the size of the array is n-1
    def missingNumber(self, n, arr):
        su=0
        m=0
        for i in arr:
            su+=i
        for i in range(0,n+1):
            m+=i
        
        return m-su
        
        # code here








#{ 
 # Driver Code Starts
#Initial Template for Python 3

t = int(input())
for _ in range(0, t):
    n = int(input())
    arr = list(map(int, input().split()))
    s = Solution().missingNumber(n, arr)
    print(s)

# } Driver Code Ends

File: Difficulty: Easy/Missing in Array/missing-in-array.java
class Solution {
    int missingNum(int arr[]) {
        // code here
        int n = arr.length;

        // Your Code Here
        int s=0,c=0;
        for(int i=1;i<=n+1;i++){
            s+=i;
        }
        for(int i = 0;i < arr.length ; i++)
        {
            c+=arr[i];
        }
        return s-c;
    }
}